### PR TITLE
add IDatabase type

### PR DESCRIPTION
**Pull request by ramirobg94**
## Description

- #### Added/Removed/Updated something...
   
- #### Added/Removed/Updated something else...
    

## Types of changes

- [ ] Docs change
- [ ] Refactoring
- [ ] Dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

#### Do you consider this PR needs further testing?

- [ ] No
- [ ] Yes

### CODE REVIEW

**Comment by minotogna** [comment]
Task linked: [CU-hf7bp3 Use arena-server DB](https://app.clickup.com/t/hf7bp3)

File: src/db/db.ts [hunk 1]
```diff
@@ -51,5 +51,6 @@ const config = ProcessEnv.dbUrl
     }
 
 export type BaseProtocol<T = any> = pgPromise.IBaseProtocol<T>
+export type IDatabase<T = any> = pgPromise.IDatabase<T>
```
Comment (sorja) [inline src/db/db.ts:54]: Should we remove the I prefix to match other?

**Review by sorja — CHANGES_REQUESTED** [review]

File: src/db/db.ts (hunk 1 above)
Comment (minotogna) [reply src/db/db.ts:54]: where this is needed ?

Comment (ramirobg94) [reply src/db/db.ts:54]: The type is needed in some comments
https://github.com/openforis/arena/pull/1468/files/af505e5b5e56630890923c47308aff25640fe40d#diff-0872008e800b44c5d35e0d94f257206bd7f4f23a325df93b9abf36768e5fb3f6R84

**Pull request closed without merging**